packages/crayons-styles/scripts/convertTokens.js

```diff
@@ -5,25 +5,32 @@ import { convertTokenToString } from '../src/utils.js';
 const tokenOutDir = './tokens/';
 const cssOutDir = './css/';
 const tokensDir = './design-tokens/';
+const globalFileName = 'crayons.json';
 
 function readFiles(rootPath) {
```

Comment:
can we split this function in to 2 , one for readingFiles and one for parsing the contents of the file to form combinedTokens and return that value

Reply:
As discussed, renamed the function name.